### PR TITLE
add rajivranjanmars

### LINKED ISSUE #496: Rajiv Ranjan Mars
image

### PR DESCRIPTION
**Pull request by rajivranjanmars**
resolve #496

### CODE REVIEW

**Review by starlightknown — APPROVED** [review]
LGTM

**Review by schmelto — APPROVED** [review]
LGTM 🚀

Edit:

Ups sorry found some styling issues :)

File: data.json [hunk 1]
```diff
@@ -874,5 +874,11 @@
     "githubUsername": "Saatvik21",
     "imageUrl": "https://user-images.githubusercontent.com/61618767/117973604-3bf81d00-b34a-11eb-86a2-8bbd7f19b6d5.png",
     "issueNumber": 493
+  },
+   {
```
Comment (schmelto) [inline data.json:878]:
```suggestion
  {
```

File: data.json [hunk 2]
```diff
@@ -874,5 +874,11 @@
...
+    "name": "Rajiv Ranjan Mars",
+    "githubUsername": "rajivranjanmars",
+    "imageUrl": "https://user-images.githubusercontent.com/67179338/118148700-7509bd80-b42e-11eb-9b52-1bd3463c4e1c.png",
+    "issueNumber": 496
   }
-]
```
Comment (schmelto) [inline data.json:878]: what have you done here?

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
Further could you resolve the merge conflict. IF you have any trouble or questions with this feel free to ask 🚀

File: data.json (hunk 2 above)
Comment (rajivranjanmars) [reply data.json]: Is it good now or I need to add some more changes?

Comment (schmelto) [reply data.json]: looks good for me :)

File: data.json [hunk 3]
```diff
@@ -875,10 +875,17 @@
...
+    "githubUsername": "rajivranjanmars",
+    "imageUrl": "https://user-images.githubusercontent.com/67179338/118148700-7509bd80-b42e-11eb-9b52-1bd3463c4e1c.png",
+    "issueNumber": 496
+  },
   {
     "name": "Ritesh Kumar",
     "githubUsername": "wordssaysalot",
     "imageUrl": "https://user-images.githubusercontent.com/56651899/118150685-71773600-b430-11eb-95f7-38a7479c5e34.png",
     "issueNumber": 498
+
```
Comment (schmelto) [inline data.json:889]:
```suggestion
```

**Review by GaryNLOL — APPROVED** [review]

**Review by JayantGoel001 — APPROVED** [review]

File: data.json (hunk 3 above)
Comment (schmelto) [inline data.json:889]: Could you remove this line please? Otherwise its looking good for me

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
Hey @rajivranjanmars did you meant to close this one? It was only one little thing more to do.

**Comment by rajivranjanmars** [comment]
I would have loved to do that but not on the cost of removing the profile views section.

**Comment by schmelto** [comment]
> I would have loved to do that but not on the cost of removing the profile views section. 

Sorry I don't understand. Why removing the profile views section?

**Comment by rajivranjanmars** [comment]
It was creeating some problem.
image

**Comment by schmelto** [comment]
> It was creeating some problem.
> image
> 

Yeah it was only a suggestion from my side.

Nothing to worry about :) you kindly closed this one and that's ok because it's your Readme!

And I really like the profile readme!

**Comment by rajivranjanmars** [comment]
Actually, I can not reopen the pull request because I had done some new changes in my profile section 
Can I restart the whole process of issue and pr again?

**Comment by schmelto** [comment]
> Actually, I can not reopen the pull request because I had done some new changes in my profile section 
> Can I restart the whole process of issue and pr again?

Sure! 🚀

**Comment by kaiwalyakoparkar** [comment]
> Actually, I can not reopen the pull request because I had done some new changes in my profile section
> Can I restart the whole process of issue and pr again?

Sure :rocket: No problem

**Pull request closed without merging**